openmrs/apps/reports/TESTING_REPORT/sql/indicator7a_not_available.sql

```diff
@@ -0,0 +1,12 @@
+SELECT 'Not available' AS 'Not available',
```

Comment:
@tmvumbi2 Please update **age** dissagregation and **gender** dissagregation as per template:

![template](https://user-images.githubusercontent.com/1198827/75423867-464a1900-5948-11ea-964b-98507fce707b.png)